ui/src/app/utils/index.tsx

```diff
@@ -510,3 +510,14 @@ export function highlightSearchTerm(searchTerm: string, stringToHighlight: strin
     </span>);
 }
 
+// render a float value as US currency, rounded to cents: 255.372793 -> $255.37
+// negative values are rendered as $0
+export function renderUSD(value: number) {
+  value = value || 0.0;
+  if (value < 0.0) {
```

Comment:
Opt nit: any reason not to just use 0 instead of 0.0 here?

Reply:
to be explicit that it's a double rather than an int.  Or maybe that is not an appropriate distinction for JS?